Dockerfile

```diff
@@ -1,13 +1,5 @@
-FROM alpine:3.4
-ADD pom.xml /
+FROM alpine:3.7
 RUN apk update \
-  && apk add tar ca-certificates openjdk8 \
-  && wget http://www-eu.apache.org/dist/maven/maven-3/3.3.9/binaries/apache-maven-3.3.9-bin.tar.gz \
-  && tar xf apache-maven-3.3.9-bin.tar.gz \
-  && ln -sf /apache-maven-3.3.9/bin/mvn /usr/bin/mvn \
-  && find /etc/ssl/certs -name "*.pem" | xargs -I {} keytool -import -alias {} -keystore /usr/lib/jvm/java-1.8-openjdk/jre/lib/security/cacerts -file {} -noprompt -storepass changeit \
-  && mvn verify \
-  && apk del tar \
-  && rm -rf /apache-maven-3.3.9-bin.tar.gz \
+  && apk add openjdk8 maven \
   && rm -rf /var/cache/apk/*
 ADD settings.xml /root/.m2/
```

Comment:
So the only thing this base has now is the settings.xml? Is it worth keeping it just for this? We already have the settings.xml [here](https://github.com/Financial-Times/nexus-settings/blob/master/public-settings.xml) and can reference it from other images.

Reply:
Yes. I agree... I think it is better to use a pure alpine directly in our java apps instead of this for a single file. In this way, we propagate the latest alpine/java8 fixes every time we upgrade a docker image.

Reply:
Once you're happy with the new approach, I suggest you re-purpose this PR (or close it and create a new one) to mark this project as deprecated in the readme, with instructions on the replacement implementation.